### PR TITLE
migrate from i2ml pr 829

### PR DESCRIPTION
**Pull request by lisa-wm**

### CODE REVIEW

**Comment by patriciomassaro** [comment]
[Fix typos noted by Viktor](https://trello.com/c/kBRHan9c/220-fix-typos-noted-by-viktor)

**Comment by patriciomassaro** [comment]
Typos noted by Viktor that I don't think we should change will be commented below

**Comment by patriciomassaro** [comment]
I don't think we should correct this, we are using the command defined in latex-math as Indicator function.

image

**Comment by patriciomassaro** [comment]
Sign is the command available in latex-math. I don't think we should change it.
image

File: slides/advriskmin/slides-advriskmin-regression-l2.tex [hunk 1]
```diff
@@ -67,7 +67,7 @@
 \begin{itemize}
 \item By the law of total expectation
   \begin{eqnarray*}
-    \riskf &=& \E_{xy} \left[\Lxy\right]
```
Comment (patriciomassaro) [inline slides/advriskmin/slides-advriskmin-regression-l2.tex:70]: image

File: slides/advriskmin/slides-advriskmin-regression-l2.tex [hunk 2]
```diff
@@ -112,15 +112,15 @@
 
 For the optimal constant model $\fx = \theta$ for the L2-loss $\Lxy = \left(y - \fx\right)^2$ we solve the optimization problem 
 $$
-\argmin_{f \in \Hspace} \riskef = \argmin_{c \in \R} \sumin (\yi - \theta)^2.
```
Comment (patriciomassaro) [inline slides/advriskmin/slides-advriskmin-regression-l2.tex:115]: It should be Theta

image

File: slides/advriskmin/slides-advriskmin-regression-l2.tex [hunk 3]
```diff
@@ -112,15 +112,15 @@
...
-\argmin_{f \in \Hspace} \riskef = \argmin_{c \in \R} \sumin (\yi - \theta)^2. 
+\argmin_{f \in \Hspace} \riskef = \argmin_{\theta \in \R} \sumin (\yi - \theta)^2. 
 $$
 
 
 We calculate the first derivative of $\riske$ w.r.t. $\theta$ and set it to $0$: 
 
 
 \begin{eqnarray*}
-\frac{\partial \risket}{\partial \theta} = 2 \sumin \left(\yi - \theta\right) &\overset{!}{=}& 0 \\
```
Comment (patriciomassaro) [inline slides/advriskmin/slides-advriskmin-regression-l2.tex:123]: image

File: slides/advriskmin/slides-advriskmin-classification-bernoulli.tex [hunk 4]
```diff
@@ -143,7 +143,7 @@
 
 \begin{eqnarray*}
   \riskf &=& \E_x \left[L(1, \fx) \cdot \eta(\xv) + L(-1, \fx) \cdot (1 - \eta(\xv)) \right] \\
-  &=&  \log(1 + \exp(- \fx)) \eta(\xv)+ \log(1 + \exp(\fx)) (1 - \eta(\xv)).
```
Comment (patriciomassaro) [inline slides/advriskmin/slides-advriskmin-classification-bernoulli.tex:146]: image

File: slides/multiclass/slides-mc-binary-reduction.tex [hunk 5]
```diff
@@ -49,7 +49,7 @@
 \begin{itemize}
 
  \item The k-th column defines how classes of all observations are encoded in the binary subproblem / for binary classifier $f_k(\xv)$.
-    \item Entry $(m, l)$ takes values $\in \{-1, 0, +1\}$
```
Comment (patriciomassaro) [inline slides/multiclass/slides-mc-binary-reduction.tex:52]: image

File: slides/cod/slides-cod-examples.tex [hunk 6]
```diff
@@ -186,7 +186,7 @@
 \begin{itemize}
 \item The regression coefficients of the noise features can not be estimated precisely as zero in the unregularized LM due to small random correlations. 
 \item With an increasing number of these noise features, the prediction error rises.
-\item To see this, we can quantify the influence of the noise features on the prediction of each iteration. \\
```
Comment (patriciomassaro) [inline slides/cod/slides-cod-examples.tex:189]: image

File: slides/regularization/slides-regu-geom-l1.tex [hunk 7]
```diff
@@ -41,7 +41,7 @@
     \item If we plug this approximation into $\mathcal{R}_{\text{reg}}(\thetab)$, the result nicely decomposes into a sum over the parameters:
   $$\mathcal{\tilde R}_{\text{reg}}(\thetab) = \mathcal{R}_{\text{emp}}(\thetah) + \sum_j \left[ \frac{1}{2} H_{j,j} (\theta_j - \hat{\theta}_j)^2 \right] + \sum_j \lambda |\theta_j|$$
   % where $\thetah$ is the minimizer of the unregularized risk $\risket$.
-    \item We can minimize analyically:
```
Comment (patriciomassaro) [inline slides/regularization/slides-regu-geom-l1.tex:44]: image

File: slides/gaussian-processes/slides-gp-prediction.tex [hunk 8]
```diff
@@ -370,7 +370,7 @@ \section{Noisy Gaussian Process}
   with 
   \begin{eqnarray*}
     \bm{m}_{\text{post}} &=& \Kmat_{*}^{T} \left(\Kmat+ \sigma^2 \cdot \id\right)^{-1}\bm{y} \\
-    \bm{K}_\text{post} &=& \Kmat_{**} - \Kmat_*^T \left(\Kmat ^{-1} + \sigma^2 \cdot \id\right)\Kmat_*,
```
Comment (patriciomassaro) [inline slides/gaussian-processes/slides-gp-prediction.tex:373]: image

File: slides/gaussian-processes/slides-gp-prediction.tex [hunk 9]
```diff
@@ -370,7 +370,7 @@ \section{Noisy Gaussian Process}
...
+    \bm{K}_\text{post} &=& \Kmat_{**} - \Kmat_*^T \left(\Kmat  + \sigma^2 \cdot \id\right)^{-1}\Kmat_*,
```
Comment (patriciomassaro) [inline slides/gaussian-processes/slides-gp-prediction.tex:373]: image

File: slides/advriskmin/slides-advriskmin-regression-l2.tex [hunk 10]
```diff
@@ -76,7 +76,7 @@
 
 \item Since $\Hspace$ is unrestricted we can choose $f$ as we wish: At any point $\xv = \xv$ we can predict any value $c$ we want. The best point-wise prediction is the conditional mean
 $$
-  \fxh = \mbox{argmin}_c \E_{y|x}\left[(y - c)^2 ~|~ \xv = \xv \right]\overset{(*)}{=} \E_{y|x} \left[y ~|~ \xv \right]. 
+  \fxbayes = \mbox{argmin}_c \E_{y|x}\left[(y - c)^2 ~|~ \xv = \xv \right]\overset{(*)}{=} \E_{y|x} \left[y ~|~ \xv \right].
```
Comment (patriciomassaro) [inline slides/advriskmin/slides-advriskmin-regression-l2.tex:79]: image

**Comment by ludwigbothmann** [comment]
> I don't think we should correct this, we are using the command defined in latex-math as Indicator function.
> 
> image

in that case I agree :+1:

File: slides/advriskmin/slides-advriskmin-classification-bernoulli.tex (hunk 4 above)
Comment (ludwigbothmann) [reply slides/advriskmin/slides-advriskmin-classification-bernoulli.tex:146]: :+1:

File: slides/advriskmin/slides-advriskmin-classification-bernoulli.tex [hunk 11]
```diff
@@ -143,7 +143,7 @@
 
 \begin{eqnarray*}
   \riskf &=& \E_x \left[L(1, \fx) \cdot \eta(\xv) + L(-1, \fx) \cdot (1 - \eta(\xv)) \right] \\
-  &=&  \log(1 + \exp(- \fx)) \eta(\xv)+ \log(1 + \exp(\fx)) (1 - \eta(\xv)). 
+  &=& \E_x \left[ \log(1 + \exp(- \fx)) \eta(\xv)+ \log(1 + \exp(\fx)) (1 - \eta(\xv)) \right]
```
[the remark this answers is not in the document]
Comment (ludwigbothmann) [reply slides/advriskmin/slides-advriskmin-classification-bernoulli.tex:146]: :+1:

File: slides/advriskmin/slides-advriskmin-regression-l2.tex (hunk 1 above)
Comment (ludwigbothmann) [reply slides/advriskmin/slides-advriskmin-regression-l2.tex:70]: :+1:

File: slides/advriskmin/slides-advriskmin-regression-l2.tex [hunk 12]
```diff
@@ -76,7 +76,7 @@
 
 \item Since $\Hspace$ is unrestricted we can choose $f$ as we wish: At any point $\xv = \xv$ we can predict any value $c$ we want. The best point-wise prediction is the conditional mean
 $$
-  \fxh = \mbox{argmin}_c \E_{y|x}\left[(y - c)^2 ~|~ \xv = \xv \right]\overset{(*)}{=} \E_{y|x} \left[y ~|~ \xv \right].
```
[the remark this answers is not in the document]
Comment (ludwigbothmann) [reply slides/advriskmin/slides-advriskmin-regression-l2.tex:79]: :+1:

File: slides/advriskmin/slides-advriskmin-regression-l2.tex (hunk 10 above)
Comment (ludwigbothmann) [reply slides/advriskmin/slides-advriskmin-regression-l2.tex:79]: :+1:

File: slides/advriskmin/slides-advriskmin-regression-l2.tex (hunk 2 above)
Comment (ludwigbothmann) [reply slides/advriskmin/slides-advriskmin-regression-l2.tex:115]: :+1:

File: slides/advriskmin/slides-advriskmin-regression-l2.tex (hunk 3 above)
Comment (ludwigbothmann) [reply slides/advriskmin/slides-advriskmin-regression-l2.tex:123]: :+1:

File: slides/cod/slides-cod-examples.tex (hunk 6 above)
Comment (ludwigbothmann) [reply slides/cod/slides-cod-examples.tex:189]: :+1:

File: slides/gaussian-processes/slides-gp-prediction.tex (hunk 8 above)
Comment (ludwigbothmann) [reply slides/gaussian-processes/slides-gp-prediction.tex:373]: :+1:

File: slides/gaussian-processes/slides-gp-prediction.tex (hunk 9 above)
Comment (ludwigbothmann) [reply slides/gaussian-processes/slides-gp-prediction.tex:373]: :+1:

File: slides/multiclass/slides-mc-binary-reduction.tex (hunk 5 above)
Comment (ludwigbothmann) [reply slides/multiclass/slides-mc-binary-reduction.tex:52]: :+1:

File: slides/regularization/slides-regu-geom-l1.tex (hunk 7 above)
Comment (ludwigbothmann) [reply slides/regularization/slides-regu-geom-l1.tex:44]: :+1:

**Review by ludwigbothmann — APPROVED** [review]

**Pull request merged**